resources/sass/app.scss

```diff
@@ -1,3 +1,223 @@
-div.title {
-	color: red;
+$color-primary: #2980b9;
+$color-dark: #333333;
+$color-white: #ffffff;
+
+.ui.secondary.pointing.menu{
+	background: $color-dark;
+	padding-top: 10px;
+	margin-bottom: 0px;
+	.item{
+		color: $color-white !important;
```

Comment:
Me parece que hay demasiados `!important` en este archivo y eso es un code smell. Are they necessary? Could you achieve the same thing with a class (maybe applied more than once)?

Reply:
listo :)